keyboards/handwired/onekey/rp2040/rp2040.c

```diff
@@ -0,0 +1,11 @@
+// Copyright 2022 Stefan Kerkmann
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include QMK_KEYBOARD_H
+
+void keyboard_post_init_user(void) {
+    debug_enable   = true;
+    debug_matrix   = true;
+    debug_keyboard = true;
+    debug_mouse    = true;
+}
```

Comment:
```suggestion
void keyboard_post_init_kb(void) {
    debug_enable   = true;
    debug_matrix   = true;
    debug_keyboard = true;
    debug_mouse    = true;
    keyboard_post_init_user();
}
```
I stumbled on this the other day and forgot to comment.